### PR TITLE
Update branch w/ new dependencies

### PR DESCRIPTION
**Pull request by StarrKiss**
I have updated all dependencies to latest version, and get a clean Gradle build and run on emulator.  All dependencies have been updated to the latest version (Including Kotlin) and nothing (Immediately) appears broken.

### CODE REVIEW

**Comment by StarrKiss** [comment]
Now targets v29 api, still would like someone to test on their own machine/real hardware

**Comment by StarrKiss** [comment]
Error fixed, issue w/ androidX

File: app/build.gradle [hunk 1]
```diff
@@ -35,8 +35,8 @@ android {
     }
 
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
+        sourceCompatibility kotlin_version
+        targetCompatibility kotlin_version
```
Comment (bherbst) [inline app/build.gradle]: These should both stay `JavaVersion.VERSION_1_8`. Kotlin still compiles down to Java bytecode, and these specify which version of Java bytecode that we generate. The `kotlin_version` variable won't mean anything in this context.

File: build.gradle [hunk 2]
```diff
@@ -1,4 +1,6 @@
-// Top-level build file where you can add configuration options common to all sub-projects/modules.
+ext {
+    var = '27.1.1'
+}// Top-level build file where you can add configuration options common to all sub-projects/modules.
```
Comment (bherbst) [inline build.gradle]: Is this `var` ever used? If not, let's get rid of it. If it is used, we should give it a better name.

**Comment by StarrKiss** [comment]
> ```
> android.useAndroidX=true
> android.enableJetifier=true
> ```

Unfortunately, doing this raises another error, as many activities extend AppCompat, and so this throws a lot of errors.  Neither of the suggested changes solve the androidX error, after searching online, it appears the only solution is to fully "upgrade" (sidegrade, if you ask me) to AndroidX

**Comment by bherbst** [comment]
There are a couple options for moving forward. I'd recommend this route:

Leave the support libraries where they were at for now (27.1.1). This also means going back to `compileSdkVersion 27`.

We can then do a separate PR to migrate to AndroidX (which we should do anyway) when we are ready, and we would still get the other library updates from this PR.

**Pull request closed without merging**